CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # HDMF Changelog
 
+## HDMF 3.5.7 (Upcoming)
+
+### New features and Minor Improvements
+- Updated `ExternalResources` to have `FilesTable` and new methods to query data. the `ResourcesTable` has been removed along with methods relating to `Resource`. @mavaylon [#850](https://github.com/hdmf-dev/hdmf/pull/850)
```

Comment:
```suggestion
- Updated `ExternalResources` to have `FileTable` and new methods to query data. the `ResourceTable` has been removed along with methods relating to `Resource`. @mavaylon [#850](https://github.com/hdmf-dev/hdmf/pull/850)
```

These tables are actually named `FileTable` and `ResourceTable`. BUT I had been thinking that to be consistent with the rest of NWB, these should be `FilesTable` and `ResourcesTable` (and also pluralize the other table names). Do you want to make that change in this PR or do it in a later one?